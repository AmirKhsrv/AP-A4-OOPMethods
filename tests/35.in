show_author_info 692
credit 38
best_reviewer
best_reviewer
best_book
recommend_first_approach 73
best_book
show_sorted_shelf 77 read Sequential Art
