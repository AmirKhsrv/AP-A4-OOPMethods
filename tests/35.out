id: 692
Name: Emery Lord
Year of Birth: 1962
Place of Birth: Bahrain
Member Since: January 20 2015
Genres: Contemporary, Young Adult
Books:
id: 0 Title: When We Collided
0.007722
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 260
Title: The Dry
Genre: Mystery
Author: Jane Harper
Reviews:
id: 691 Rating: 4 Likes: 0 Date: September 25 2019
id: 1424 Rating: 0 Likes: 0 Date: September 6 2017
id: 1896 Rating: 5 Likes: 0 Date: July 7 2018
id: 3139 Rating: 5 Likes: 0 Date: January 20 2015
id: 3367 Rating: 0 Likes: 0 Date: April 9 2014
id: 3705 Rating: 0 Likes: 0 Date: August 24 2018
id: 4488 Rating: 4 Likes: 11 Date: April 2 2017
id: 5500 Rating: 0 Likes: 0 Date: March 23 2019
id: 6249 Rating: 2 Likes: 4 Date: August 2 2015
id: 6435 Rating: 4 Likes: 0 Date: May 17 2016
id: 7541 Rating: 5 Likes: 4 Date: November 20 2017
id: 7583 Rating: 1 Likes: 0 Date: October 12 2017
id: 7882 Rating: 2 Likes: 1 Date: June 5 2018
id: 7914 Rating: 4 Likes: 0 Date: March 22 2016
id: 8970 Rating: 3 Likes: 0 Date: December 1 2017
id: 8991 Rating: 1 Likes: 0 Date: April 5 2018
id: 9233 Rating: 2 Likes: 0 Date: November 1 2018
id: 9366 Rating: 2 Likes: 0 Date: July 5 2010
id: 9535 Rating: 4 Likes: 0 Date: August 10 2019
id: 9770 Rating: 4 Likes: 0 Date: February 13 2014
id: 10148 Rating: 5 Likes: 2 Date: May 8 2018
id: 10555 Rating: 5 Likes: 0 Date: December 27 2013
id: 10711 Rating: 3 Likes: 12 Date: May 22 2018
id: 10774 Rating: 2 Likes: 50 Date: July 26 2014
id: 10781 Rating: 2 Likes: 18 Date: November 12 2016
id: 11077 Rating: 1 Likes: 90 Date: September 14 2019
id: 12171 Rating: 5 Likes: 1 Date: May 15 2019
id: 12348 Rating: 2 Likes: 0 Date: April 27 2018
id: 13595 Rating: 5 Likes: 0 Date: August 20 2015
id: 14466 Rating: 2 Likes: 0 Date: August 25 2010
id: 14597 Rating: 3 Likes: 0 Date: August 29 2018
id: 15113 Rating: 1 Likes: 0 Date: February 29 2017
id: 15182 Rating: 3 Likes: 0 Date: March 22 2018
id: 15585 Rating: 5 Likes: 0 Date: February 23 2018
id: 16008 Rating: 4 Likes: 1 Date: March 9 2008
id: 16427 Rating: 4 Likes: 0 Date: September 2 2019
id: 17247 Rating: 5 Likes: 0 Date: January 18 2014
id: 17394 Rating: 4 Likes: 0 Date: May 13 2019
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 354
Title: Awaken Me
Genre: Fantasy
Author: E.M. Sky
***
id: 107
Title: Finger Lickin' Fifteen
Genre: Mystery
Author: Janet Evanovich
***
id: 184
Title: Never Deal with a Dragon
Genre: Fantasy
Author: Robert N. Charrette
***
id: 379
Title: Ravage Me
Genre: Romance
Author: Ryan Michele
***
id: 299
Title: Switch! The Lost Kingdoms of Karibu
Genre: Fantasy
Author: Karen  Prince
***
id: 54
Title: The 19th Wife
Genre: Historical
Author: David Ebershoff
***
id: 110
Title: The Shape of Mercy
Genre: Historical
Author: Susan Meissner
***
id: 192
Title: The Sumerton Women
Genre: Historical
Author: D.L. Bogdan
***
id: 263
Title: Timepiece
Genre: Young Adult
Author: Myra McEntire
***
id: 377
Title: West of Eden
Genre: Science Fiction
Author: Harry Harrison
***
